magic/core/src/main/java/link/magic/android/core/relayer/WebViewWrapper.kt

```diff
@@ -33,7 +33,7 @@ import java.util.*
 /**
  * This class is designed to be instantiate only once. As for the webview
  */
-class WebViewWrapper internal constructor(context: Context, private val urlBuilder: URLBuilder) {
+class WebViewWrapper internal constructor(context: Context, private val urlBuilder: URLBuilder, resetContext: () -> Unit) {
 
     internal val mMutableContext = MutableContextWrapper(context)
     private val mImmutableContext = context
```

Comment:
Correct me if I'm wrong, I assume this mImmutableContext is the application context stored as a copy when instantiating a webview. I think we can just reset to this context.

Reply:
Yes, and we do.